dev/com.ibm.ws.microprofile.health.2.0_fat/bnd.bnd

```diff
@@ -29,18 +29,16 @@ tested.features=servlet-4.0,cdi-2.0,mphealth-3.0,mpConfig-1.4,mpConfig-2.0
 	com.ibm.websphere.javaee.jsonp.1.1;version=latest, \
 	com.ibm.websphere.javaee.cdi.1.2;version=latest, \
 	com.ibm.websphere.org.eclipse.microprofile.health.2.0;version=latest, \
-    com.ibm.websphere.javaee.servlet.3.1;version=latest, \
-    com.ibm.websphere.org.eclipse.microprofile.reactive.messaging.1.0;version=latest,\
+	com.ibm.websphere.javaee.servlet.3.1;version=latest, \
+	com.ibm.websphere.org.eclipse.microprofile.reactive.messaging.1.0;version=latest,\
 	com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0;version=latest,\
 	com.ibm.websphere.org.eclipse.microprofile.config.1.3;version=latest,\
 	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
 	com.ibm.websphere.javaee.cdi.2.0;version=latest,\
 	com.ibm.websphere.javaee.jsonb.1.0;version=latest,\
 	com.ibm.ws.org.slf4j.api.1.7.7;version=latest,\
```

Comment:
Can this be removed as well? I'm not sure if there's anything extra in com.ibm.ws.org.slf4j.api, but I don't want it to cause issues if testcontainers moves to a new version of slf4j